posts/viewsfolder/feed_views.py

```diff
@@ -141,7 +147,7 @@ def get(self, request, authorid):
             serializer = PostSerializer(result_page, many=True)
         else:
             # TODO Get this working with external friending
-            posts = Post.objects.filter(author=authorid)
+            posts = Post.objects.filter(author=authorid).order_by("-published")
```

Comment:
Sorting is not in spec. Leave it out for now

Reply:
Ill remove it, but theres a note in the forum how posts should be ordered by date with github posts, also all other views are sorted atm.